ocs_ci/framework/conf/default_config.yaml

```diff
@@ -120,7 +120,7 @@ ENV_DATA:
 # This section is related to upgrade
 UPGRADE:
   upgrade_to_latest: true
-  ocp_channel: "stable-4.4"
+  ocp_channel: "stable-4.5"
```

Comment:
I don't think such channel exists when OCP 4.5 is not GAed yet.

@AviadP can tell you more details what to use but I guess we should use something similar what we had here:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocp_version/ocp-4.4-candidate-upgrade.yaml 

Most likely there will be already candidate build for 4.5 but haven't checked here.

Other option to use nightly like in this config:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocp_version/ocp-4.4-upgrade.yaml

Which will be probably better option, but for this you will need to change also one line below.

Reply:
Please check the code how it is used or wait for @AviadP explanation. I do not remember it either. Just brought this up for discussion as we need to set it correctly for OCP upgrade.

Reply:
@petr-balogh, channel "cnadidate-4.5" should be ready for use. as far as i understand, "stable-4.5" isnt
i will create config files for 4.5-nightly or 4.5 candidate

Reply:
`ocp_channel` is now set to `candidate-4.5`.